applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -6,7 +6,7 @@
 @import views.support.{RenderClasses, SeoOptimisedContentImage, StripHtmlTags, Video640}
 
 @defining(page.media, page.media match { case _: Audio => "audio" case _ => "video" }){ case (media, mediaType) =>
-<div class="l-side-margins l-side-margins--media">
+<div class="l-side-margins @if(!page.media.commercial.isAdvertisementFeature){l-side-margins--media}">
```

Comment:
Yeah, same with my css changes. There should really be a glabs media tone created separately, rather than loads of overrides, as glabs media pages are a different layout and completely different colour scheme to media.